src/endToEnd.ts

```diff
@@ -12,7 +12,13 @@ export default async function encryptAndGetTracker(
       '/electionguard/EncryptBallot',
       {
         method: 'post',
-        body: JSON.stringify(votes),
+        body: JSON.stringify({
+          ballot: votes,
```

Comment:
![image](https://user-images.githubusercontent.com/28444/73499823-00fcf080-4376-11ea-8267-fdde4d513182.png)

Reply:
😂 Sorry, couldn't help it.

Reply:
lol nice :)